accounts/utils.py

```diff
@@ -0,0 +1,80 @@
+from datetime import timedelta
```

Comment:
Maybe a more specific filename like `ipc.py` so they import it with `django_labs_accounts.ipc`

Reply:
Agreed